src/ai/ai-bedrock-cohere-embedding-model.ts

```diff
@@ -0,0 +1,15 @@
+export interface CohereEmbeddingBody {
+  texts: string[];
+  input_type:
+    | 'search_document'
+    | 'search_query'
+    | 'classification'
+    | 'clustering';
+}
+
+export interface CohereEmbeddingResponse {
+  id: string;
+  texts: string[];
+  embeddings: number[][];
```

Comment:
is there any explanation on why there might be multiple embeddings?

I know openAI has multiple embeddings when you send it multiple chunk at once, but it seems by your code here that you can only send one string per request, so having multiple embeddings as an output is odd

Reply:
@Mael-Abgrall, we can send multiple chunks at once but all of them have the same chunkID